src/components/Contacts/ContactDetails/ContactDetailsTab/Tags/ContactTags.tsx

```diff
@@ -84,6 +84,22 @@ export const ContactTags: React.FC<ContactTagsProps> = ({
     }
   };
 
+  const handleValidation = async ({
+    tagList,
+  }: {
+    tagList: string[] & never[];
+  }): Promise<void> => {
+    for (let i = 0; i < tagList.length; i++) {
+      if (contactTags.includes(tagList[i])) {
+        enqueueSnackbar(t('Cannot add duplicate tags'), {
+          variant: 'error',
+        });
+        tagList.pop();
+      }
+    }
+    return;
```

Comment:
Typically in JavaScript, When you want to edit any array, you create a new array from the original array and make edits on the new array, so you don't edit the original.

Why do you need to remove items from the tagList? Rather than just flagging to the user that they can't have dup tags?

Reply:
If you want to remove the tags, I would update the message to state that you removed the dup tag, so the user knows.